relayers.json

```diff
@@ -142,5 +142,24 @@
                 "networkId": 42
             }
         ]
+    },
+    {
+        "name": "Shark Relay",
+        "url": "https://sharkrelay.com",
+        "networks": [
+            {
+                "networkId": 1,
+                "sra_http_endpoint": "https://api.sharkrelay.com/0x/v0/",
```

Comment:
This doesn't appear to be working. Can you verify that this is SRA-compliant? Check out this tool: https://github.com/0xProject/0x-monorepo/tree/development/packages/sra-report